packages/holocron/src/moduleRegistry.js

```diff
@@ -30,8 +31,12 @@ function getModuleBlockList() {
   return moduleBlockList;
 }
 
+function addHigherOrderComponent(module) {
+  return holocronModule(module.holocron)(module);
+}
+
 function registerModule(moduleName, module) {
-  modules = modules.set(moduleName, module);
+  modules = modules.set(moduleName, addHigherOrderComponent(module));
```

Comment:
`registerModule` is only called on the clientside and we use the decorator here.